package.json

```diff
@@ -28,7 +28,7 @@
 		"url": "https://hieunguyen.info/"
 	},
 	"contributors": [
-    "Kekschen (https://github.com/Kek5chen)",
```

Comment:
something in the production line produced this. no idea whats the matter with that